### ISSUE TITLE
Responsive support for dashboard?

### ISSUE THREAD

**Issue by phanan** [fzaninotto/uptime#297]
The current dashboard is built on Bootstrap (v2 originally, and v3 with [my fork](https://github.com/phanan/uptime)), which means responsive support should be easy if not out of the box. Problem is, the way the datetime navigator is currently structured -- `.btn-group` mixing with a dynamically-created progressbar -- makes things ugly and even wrong on a smaller screen:

![image](https://cloud.githubusercontent.com/assets/8056274/5913109/444c82d8-a619-11e4-8a51-951e69c0aac4.png)

Anyone has an idea how to fix this? I'd be happy to implement the solution.

**Comment by Soullivaneuh** [comment]
Hard to figure a nice solution with this buttons...

Maybe single line table with table-responsive div wrapping them?

**Comment by phanan** [comment]
Hmm... `div.table-responsive` is not a bad idea actually. But then we'll need to implement at least 2 of them: the one wrapping those `btn`, and another for the progress bar above it. 

Then, because the bar actually corresponds to the charts underneath it, we may need to wrap them into `div.table-responsive` as well... And how do (or should) we sync them when user scrolls horizontally?

![image](https://cloud.githubusercontent.com/assets/8056274/5921196/3f3fb314-a67d-11e4-8c23-69d60b043d71.png)

Final concern: If users need to scroll anyway, what's the point of a "responsive" layout? 
Anyway I'll find some time to implement this and let you guys know.

**Comment by Soullivaneuh** [comment]
> Final concern: If users need to scroll anyway, what's the point of a "responsive" layout? 

Well, I agreed with that. But how do you want to render responsive a line of 26 buttons without some scroll ? :)

**Comment by phanan** [comment]
> Well, I agreed with that. But how do you want to render responsive a line of 26 buttons without some scroll ? :)

Exactly. But is it a must that we use buttons? Is there another way to achieve the same purpose without them?

**Comment by Soullivaneuh** [comment]
@phanan what is your proposition?

**Comment by phanan** [comment]
I don't have any :(... that's why I raised this question up. Then again, will try your solution and see how well it goes.

**Comment by brantje** [comment]
I've tried something, its far from perfect.
https://github.com/brantje/uptime/tree/responsive

**Comment by elboletaire** [comment]
Sooo... still not being responsive? :cry:
